integration-tests/bulkExport.test.ts

```diff
@@ -6,6 +6,7 @@ import BulkExportTestHelper, { ExportStatusOutput } from './bulkExportTestHelper
 import { getFhirClient } from './utils';
```

Comment:
just reformatting here right?

Reply:
It's mostly reformatting. I'm setting the jest timeout for all tests in this file on line 9, instead of passing that value to each test function. As a result the indentation of the lines are changed.